### ISSUE TITLE
Unified titlebar is not working with bookmark navigation mode

### ISSUE THREAD

**Issue by dangh** [raphamorim/rio#786]
Hello,

I would love to have the unified titlebar but looks like it only work with NativeTab mode. Can we support Bookmark mode too?

Thank you.

**Comment by raphamorim** [comment]
Hey @dangh thanks for the issue,

Would you like to have titlebar on macos but using bookmark?

**Comment by dangh** [comment]
Yes @raphamorim. Unless it's intended having both titlebar and booking would be lovely.

**Comment by dangh** [comment]
Yes @raphamorim. Unless it's intended having both titlebar and booking would be lovely.

**Comment by raphamorim** [comment]
Bookmark mode was removed in the island-tabs rewrite (#1706). If unified titlebar plus the new tab mode still conflicts on macOS, please open a fresh issue.
